druid/src/widget/flex.rs

```diff
@@ -281,3 +336,62 @@ impl CrossAxisAlignment {
         }
     }
 }
+
+impl MainAxisAlignment {
```

Comment:
This looks good to my brain compiler, and I really like this style of spinning out straight logic.